src/sass/layout/_footer.scss

```diff
@@ -0,0 +1,208 @@
+.footer {
+  background-color: getColor('green-main');
+  padding: 0 20px 24px 20px;
+  @media (min-width: 768px) {
+    padding: 0;
+  }
+  @media (min-width: 1200px) {
+    padding: 0 16px 0 16px;
+  }
+}
+
+.footer-box {
+  @media (min-width: 768px) {
```

Comment:
should be there padding-bottom as for the desktop version?
for tablet - 32px;
for 480px and mobile - 40px

Reply:
there are some margins